manufacturers/rules.stanza

```diff
@@ -20,6 +20,8 @@ public pcb-rules bay-area-circuits-std-rules :
   min-silkscreen-text-height = 0.75 ; 29.5 mil (guess)
   solder-mask-registration = 0.106 ; 4 mil
   min-th-pad-expand-outer = 0.2032 ; 8 mil
+  min-soldermask-opening = 0.152
```

Comment:
Add `; TODO: lookup actual value` or something in here so we don't forget to update these with real values, otherwise we'll have designs pass verification that not invalid!

Reply:
It should be the real values

Reply:
It is incorrect for at least JLCPCB

https://jlcpcb.com/capabilities/Capabilities

Reply:
Thanks